crates/smartmodule-development-kit/src/test.rs

```diff
@@ -157,7 +158,10 @@ impl TestCmd {
 
         let test_records: Vec<Record> = test_data.into();
 
-        let output = chain.process(SmartModuleInput::try_from(test_records)?, &metrics)?;
+        let output = chain.process(
+            SmartModuleInput::try_from_records(test_records, DEFAULT_SMARTENGINE_VERSION)?,
```

Comment:
I think that you can call `set_base_timestamp` here to add current timestamp to tests

Reply:
Great! Thanks!